src/SFA.DAS.Employer.PR.Web/Models/ChangePermissionsViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace SFA.DAS.Employer.PR.Web.Models;
+
+public class ChangePermissionsViewModel : ChangePermissionsSubmitViewModel, IBackLink
```

Comment:
What is the point of IBackLink? I see the back link are physically added into the respective views

Reply:
you're right, removed